tests/unit/tasks/test_views.py

```diff
@@ -11,7 +11,6 @@ def test_task_visibility(self):
         self.assertTrue(self.client.login(username="bob", password="secret"))
         response = self.client.get(url, follow=True)
         self.assertContains(response, self.published_task1.title)
-        self.assertNotContains(response, self.unpublished_task1.title)
```

Comment:
This test should be a bit more elaborate:
* it should assert that both published and unpublished tasks are displayed
* it should assert that the unpublished task is displayed using the other style